Core/ALife.Core/ImportExport/AgentCodeSerializer.cs

```diff
@@ -167,5 +167,21 @@ private static string ProximityClusterDefinitionAsCode(ProximityCluster pc)
             return sb.ToString();
         }
 
+        public static string ConvertDoubleArrayToString(double[] inputDoubles)
+        {
+            byte[] asBytes = new byte[inputDoubles.Length * sizeof(double)];
+            Buffer.BlockCopy(inputDoubles, 0, asBytes, 0, asBytes.Length);
+            string s = Convert.ToBase64String(asBytes);
+
+            return s;
+        }
+
+        public static double[] ConvertStringToDoubleArray(string str)
+        {
+            byte[] inputBytes = Convert.FromBase64String(str);
+            double[] output = new double[inputBytes.Length / 8];
+            Buffer.BlockCopy(inputBytes, 0, output, 0, inputBytes.Length);
+            return output;
+        }
```

Comment:
More a logistical question...in cases like this, do we want to move these to the common utilities namespace? These things, for example, might be handy in the future in other cases.

Reply:
That would be a good idea for if these are ever reused